src/Services/ConvertToQtiService.php

```diff
@@ -2,36 +2,41 @@
 
 namespace LearnosityQti\Services;
 
-use LearnosityQti\AppContainer;
+use DOMDocument;
+use DOMElement;
+use Exception;
 use LearnosityQti\Converter;
 use LearnosityQti\Domain\JobDataTrait;
-use LearnosityQti\Exceptions\MappingException;
+use LearnosityQti\Processors\IMSCP\Entities\File;
+use LearnosityQti\Processors\IMSCP\Entities\ImsManifestMetadata;
+use LearnosityQti\Processors\IMSCP\Entities\Manifest;
+use LearnosityQti\Processors\IMSCP\Entities\Resource;
 use LearnosityQti\Processors\QtiV2\Out\Constants as LearnosityExportConstant;
-use LearnosityQti\Utils\AssetsFixer;
-use LearnosityQti\Utils\AssumptionHandler;
-use LearnosityQti\Utils\CheckValidQti;
-use LearnosityQti\Utils\ResponseProcessingHandler;
 use LearnosityQti\Utils\General\FileSystemHelper;
-use LearnosityQti\Utils\General\StringHelper;
+use LearnosityQti\Utils\UuidUtil;
+use LearnosityQti\Utils\General\CopyDirectoreyHelper;
```

Comment:
Remove unused use statement.

Reply:
Done

Reply:
I'm still seeing a use statement for `CopyDirectoreyHelper` above - has this change been pushed?

Reply:
Done